Add writing guidelines

stickies-v opened the pull request:
Condensed my observations from the first cohort into these guidelines. Open for comments/suggestion!

adamjonas commented on writing-guidelines.md:
```diff
@@ -0,0 +1,146 @@
+
+### 4.2. The Qala target audience
+
+The Qala program has a very clear technical focus. Most of your writing should reflect that. Even though articles catered for a general audience can be helpful (and are important for the Bitcoin ecosystem in general), these will often not be the best choice for two reasons:
```
```suggestion
The Qala program has a very clear technical focus. Most of your writing should reflect that. Even though articles catered to a general audience can be helpful (and are important for the Bitcoin ecosystem in general), these will often not be the best choice for two reasons:
```

adamjonas commented on writing-guidelines.md:
```diff
@@ -0,0 +1,146 @@
+
+When presenting an idea that is not yours, you'll have to choose between using your own wording or quoting/paraphrasing the original author. You should always strive to rephrase a topic in your own words (e.g. by phrasing from it an alternative perspective, or in simpler terms, or by combining it with other topics). Sometimes rephrasing is not possible or sensible. In that case, it is okay to quo ...
+
+Even though plagiarism is often not illegal, it is almost guaranteed to blemish your reputation if and when it is discovered. Especially in an open-source community such as Bitcoin's, be generous with your references. It ensures the original author receives credit for their work where it is due. As an additional upside, ample linking to high-quality sources is a great way to make your own article ...
```
Maybe add that is shows a laziness because it takes credit for other's work without creating value.

stickies-v replied:
Fixed in 50ed939, thanks!

adamjonas reviewed (commented):
well done

stickies-v commented on writing-guidelines.md:
```diff
@@ -0,0 +1,146 @@
+
+If you define your audience too broadly, you will upset your actual audience as you would be when reading "The Bitcoin Script Expert". Conversely, if you define your audience too narrowly, you may lose some readers that otherwise would have benefited from your writing, but are lacking just a bit too much background knowledge to follow along.
+
+Defining your audience and finding a good balance between assuming not too much and not too little pre-existing knowledge is a delicate exercise, but one that needs to be done anyway.
```
Fixed in 0d06ec6, thanks!

stickies-v commented:
Thank you both for your review, I've incorporated all suggestions and also added some more of my own. I'll leave the PR up for another day or so and if no further comments I'll merge, we can always improve later.
